airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -462,13 +462,11 @@ public QuarantinedInformation getQuarantinedInformation() {
    * delete
    */
   private Boolean skipScheduling() {
-    return workflowState.isSkipScheduling() || workflowState.isDeleted() || workflowState.isUpdated() ||
-        (!workflowState.isResetWithScheduling() && workflowState.isResetConnection());
```

Comment:
This skipScheduling logic has been simplified now that we do not need the `isResetConnection` and `isResetWithScheduling` flags - instead we now just set the `skipScheduling` flag on the ConnectionUpdaterInput when we need to skip scheduling on the next run.